src/tests/ftest/util/dmg_utils.py

```diff
@@ -206,3 +241,123 @@ def storage_reset(hosts, user=None, hugepages="4096", insecure=True):
         return None
 
     return result
+
+
+def pool_create(path, test_object, host_port, scm_size, insecure=True,
+                group=None, user=None, acl_file=None, nvme_size=None,
+                ranks=None, nsvc=None, sys=None):
+    """Execute pool create command through dmg tool to servers provided.
+
+    Args:
+        path (str): Path to the directory of dmg binary.
+        test_object (TestWithServers): Test object that calls this method.
+        host_port (list of str): List of Host:Port where daos_server runs.
+            Use 10001 for the default port number. This number is defined in
+            daos_avocado_test.yaml
+        scm_size (str): SCM size value passed into the command.
+        insecure (bool, optional): Insecure mode. Defaults to True.
+        group (str, otional): Group with priviledges. Defaults to None.
+        user (str, optional): User with priviledges. Defaults to None.
+        acl_file (str, optional): Access Control List file path for DAOS pool.
+            Defaults to None.
+        nvme_size (str, optional): NVMe size. Defaults to None.
+        ranks (str, optional): Storage server unique identifiers (ranks) for
+            DAOS pool
+        nsvc (str, optional): Number of pool service replicas. Defaults to
+            None, in which case 1 is used by the dmg binary in default.
+        sys (str, optional): DAOS system that pool is to be a part of. Defaults
+            to None, in which case daos_server is used by the dmg binary in
+            default.
+
+    Returns:
+        Avocado CmdResult object that contains exit status, stdout information.
```

Comment:
Since the rest of the docstring is following the gogole-style (https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), the `Returns:` section should be in this format:
`    <type>: <description>`.  Same comment for the other two functions below.

Reply:
Okay. Thanks for letting me know.